Readme.md

```diff
@@ -1,3 +1,21 @@
+## rust-analyzer dashboard
+
 Stores performance metrics for rust-analyzer.
 
 Metrics are automatically published by GitHub Actions.
+
+## Develop locally
+
+```sh
+# install dev-dependencies:
+npm install
```

Comment:
Shouldn't this be `npm ci` to respect the lockfile?

Reply:
The lockfile is in the `.gitignore` file. Also, these instructions are for developers, not for CI.

Reply:
Is there any reason to ignore the lockfile? In a Rust application it would be checked in and used.

Reply:
And even as a dev, would I really want to get a different version of Plotly than what the site is using? It sounds like a recipe for problems.

Reply:
Yeah you're right. ATM plotly is loaded with a `<script>` tag. The npm dependency is just to get the typescript types. I didn't look at the version, and it seems they're actually incompatible, so it's pure luck that I didn't get any errors! Also, I'm confused about the version numbers of `@types/plotly`, which don't seem to match the versions of `plotly` at all?

Regarding the lock file: After thinking about again, I agree that it should be tracked by git.

Reply:
Can you also update the instructions to `npm ci`? And maybe use the plotly package?

Reply:
Done.